app/Support/PluginManager.php

```
@@ -15,13 +15,17 @@
 {
     private const MAX_UPLOAD_BYTES = 104857600; // 100 MB
+    private const PENDING_UPDATE_PREFIX = '.pinakes-plugin-update-';
     private mysqli $db;
     private string $pluginsDir;
     private string $uploadsDir;
     private HookManager $hookManager;
     private ?string $cachedEncryptionKey = null;
     private bool $encryptionKeyResolved = false;
 
+    /** @var array<int,true> Plugins rolled back after their replacement class was loaded. */
+    private array $skipPluginIdsThisRequest = [];
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $skipPluginIdsThisRequest. Keep variable name length under 20. (undefined)

(LongVariable)